formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/Handler.scala

```diff
@@ -73,13 +74,16 @@ trait FormstackHandler[Req, Res] extends LazyLogging {
   }
 }
 
-object Handler {
+object Handler extends LazyLogging{
 
   val stage: String = sys.env.getOrElse("STAGE", "CODE") match {
     case "PROD" => "PROD"
     case _      => "CODE"
   }
-  val formstackService = new FormstackService()
+  def formstackService = if (stage == "PROD") new FormstackService() else {
+    logger.info(s"using mocked formstack service as stage is $stage")
+    FormstackServiceStub.withSuccessResponse
+  }
```

Comment:
this is not the best, but until we get some sort of Formstack sandbox environment we are just using the stub we had for tests in non `PROD` environments